.docker/ci/Dockerfile

```diff
@@ -26,12 +20,26 @@ RUN \
     apt-get -qq install -y \
         # Some source builds require a package.xml be downloaded via wget from an external location
         wget \
+        # Version control
+        git \
         # Required for rosdep command
         sudo \
         # Preferred build tools
-        python-catkin-tools \
-        clang clang-format-3.9 clang-tidy clang-tools \
-        ccache && \
+        clang clang-format-10 clang-tidy clang-tools \
+        ccache \
+        #
+        # June 2020 workaround until catkin_tools fixed https://github.com/catkin/catkin_tools/issues/603
+        # TODO: re-enable this install method python3-catkin-tools \
+        python3-pip && \
+    #
+    # TODO: remove this workaround
+    pip3 install -U catkin_tools && \
```

Comment:
That's not actually what I meant in https://github.com/catkin/catkin_tools/issues/603#issuecomment-634617492.

This would still install the released `catkin_tools`, not the version in the repository. Afaik, only the repository contains the fixes necessary to install and make `catkin_tools` work on Focal.

Reply:
fixed